examples/experimental/Federated Learning with Grid Nodes (Before remove sleep).ipynb

```diff
@@ -0,0 +1,316 @@
+{
```

Comment:
* Rename this file to: "Federated Learning with Grid Nodes.ipynb"
* Link the original PySyft tutorial that this is based on

Reply:
Done! 9e90bea